bin/test/test_helper.rb

```diff
@@ -1,9 +1,11 @@
-require_relative '../../../shared/test/common_test_helper'
+require_relative '../../shared/test/test_helper'
```

Comment:
why did this move from `bin/i18n/test` to `/bin/test`

Reply:
I just do not see any reason for creating separate `test` for each folder inside `bin`.
It would be more correct to have only one `test` folder in the root directory to test both `lib` and `bin`

Reply:
I think this move makes sense. Setting up the tests in the root directory will also enable test writing for the other scripts in this directory.

I also think it would be nice if each script had its own standalone setup, but I would like to lean towards making it easier to write tests since there doesn't seem to be a good practice of that.